ISISComputingGroup/EPICS-ioc
# Ticket1425 common calibration dir

### Description of work

Put all common calibration data in a common place under git. Follow release notes to get calibration data.

### To test

https://github.com/ISISComputingGroup/IBEX/issues/1425

See pull requests:
https://github.com/ISISComputingGroup/EPICS-eurotherm2k/pull/1
https://github.com/ISISComputingGroup/EPICS-barndoors/pull/2
https://github.com/ISISComputingGroup/ibex_gui/pull/304

I merged onto master not a branch sorry but look at latest commit:
https://github.com/ISISComputingGroup/EPICS-csm


### Acceptance criteria

The following IOC still work for conversion of data:

1. Barndoors (motor record gap to volatge)
1. Eurotherm (sensor to temp and ramps)
    1. NB Made address, not just lesser and greater (as per standup)
1. Danfysik (magnet to current)
    1. Obeys sim rec and disable macros now.
1. Issue carried over from #1403 has been resolved; see Adrian's note for ticket [#1425](https://github.com/ISISComputingGroup/IBEX/issues/1425)

#### Code Review


### Functional Tests


### Final steps
